microsoft/onefuzz
# [Breaking Change] Deploy with scriban only turned on

## Summary of the Pull Request

_What is this about?_

Closes #2767

Updates our bicep template to enable ScribanOnly feature flag. This will enforce that notification templates are only rendered with scriban.

## Info on Pull Request

_What does this include?_

* Better logging on feature flag state
* Updates to bicep template to enable ScribanOnly feature flag at deploy time

## Validation Steps Performed

_How does someone test & validate?_

Deploying this PR should create a new ScribanOnly feature flag in App Configuration (if it didn't already exist) and set the flag to enabled (if it wasn't already set to enabled).
